app/views/articles/show.html.erb

```diff
@@ -65,11 +65,11 @@
 
   <main id="main-content" class="crayons-layout__content grid gap-4">
     <div class="article-wrapper">
-      <% if !@article.published %>
+      <% unless @article.current_state == :published %>
```

Comment:
Reading note: how is `@article.published?` different from `@article.current_state == :published`?

Reply:
`published?` checks if the field `published` is true. Scheduled articles have `published` set to true too, but they have `published_at` in the future. Maybe `current_state == :published` is not beautiful, we can have another method :D

Reply:
For now this is alright; there's a future potential refactor.  The goal is to get the idea out there.

Reply:
Cool, I've refactored a little bit after your suggestion about `ActiveSupport::StringInquirer`